packages/libs/eda/src/lib/core/components/visualizations/implementations/VolcanoPlotVisualization.tsx

```diff
@@ -75,15 +81,19 @@ function createDefaultConfig(): VolcanoPlotConfig {
     log2FoldChangeThreshold: DEFAULT_FC_THRESHOLD,
     significanceThreshold: DEFAULT_SIG_THRESHOLD,
     markerBodyOpacity: 0.5,
+    independentAxisRange: undefined,
+    dependentAxisRange: undefined,
   };
 }
 
 export type VolcanoPlotConfig = t.TypeOf<typeof VolcanoPlotConfig>;
-
+// eslint-disable-next-line @typescript-eslint/no-redeclare
```

Comment:
was the second declaration giving you issues? Or is this to clean up warnings?

Reply:
Just cleaning up ESLint warnings, now that VSCode screams at me! 😂